packages/frontend/model/extract-nav.ts

```diff
@@ -19,7 +19,7 @@ const getLink = (data: {}, { isPillar }: { isPillar: boolean }): LinkType => {
 const rrLinkConfig = 'config.readerRevenueLinks';
 const buildRRLinkCategories = (
     data: {},
-    el: string,
+    el: ReaderRevenueLinkNames,
```

Comment:
Maybe a better name than 'el' here? I read it as element but was slightly confused as it's not an actual HTMLElement type.